projects/plugins/jetpack/_inc/client/state/site-products/reducer.js

```diff
@@ -77,3 +77,20 @@ export function getSiteProductMonthlyCost( state, slug ) {
 
 	return Math.ceil( ( price / 12 ) * 100 ) / 100;
 }
+
+/**
+ * Returns the discount of a product price. It bases the discount on intro offers with a yearly interval
+ *
+ * @param   {object} state - Global state tree
+ * @param   {string} slug  - Product slug
+ * @returns {number}  Discount of a product price or 0 if there is no discount
```

Comment:
Nitpick: not sure if the intention was to align the @returns description to the param names, but there's an extra space after the type.